src/validation.cpp

```diff
@@ -1050,6 +1054,21 @@ CoinsViews::CoinsViews(
     m_cacheview.reset(new CCoinsViewCache(m_catcherview.get()));
 }
 
+CChainState::CChainState(
+    /* parameters forwarded to CoinsViews */
+    size_t cache_size_bytes,
+    bool in_memory,
+    bool should_wipe,
+    std::string leveldb_name
+    // NOTE: for now m_blockman is set to a global, but this will be changed
+    // in a future commit.
+    ) : m_blockman(g_blockman)
+{
+    m_coins_views.reset(new CoinsViews(
```

Comment:
In commit "refactor: have CCoins* data managed under CChainState" (eed4e62ad6b162ff0d3e2b87eb8762a689678bca)

Suggest `x = MakeUnique<X>()` instead of `x.reset(new X())` to avoid raw pointers and new in C++, which are suspicious.